### ISSUE TITLE
[Bug]: "Unable to fetch transaction" log spam on LN log when starting up Server

### ISSUE THREAD

**Issue by Goro2030** [btcpayserver/btcpayserver#4935]
### What is your BTCPay version?

I'm running the latest version BTCPay Server v1.9.2+6193835ea.

### How did you deploy BTCPay Server?

Docker installation

### What happened?

After the upgrade to BTCPay Server v1.9.2+6193835ea, the LND log is filled with this errors:

`2023-04-28 20:03:59.331 [ERR] LNWL: unable to fetch transaction xxxxxxxxxxx for mempool: -5: No such mempool transaction. Use -txindex or provide a block hash to enable blockchain transaction queries. Use gettransaction for wallet transactions.
`

Hundreds of them.

When the node is finished starting up, RPC is up and running, then the error ceases for a while, but then resumes with less TX's ( currently 5).

I'm running the server with a Pruned node. 

Is this normal behavior, and if so, why? May this be because my LN opening channel TX's are old enough to have fallen outside of the blockchain window set in the PRUNE window?

### How did you encounter this bug?

Checked Logs

### Relevant log output

```shell
2023-04-28 20:14:53.258 [ERR] LNWL: unable to fetch transaction xxxxxxxxxxxx from mempool: -5: No such mempool transaction. Use -txindex or provide a block hash to enable blockchain transaction queries. Use gettransaction for wallet transactions.
```

### What browser do you use?

Any

### Additional information

N/a

### Are you sure this is a bug report?

- [X] I confirm this is a bug report

**Comment by dennisreimann** [comment]
This seems to be an issue related to the LND v0.16.1 update, see lightningnetwork/lnd#7640

We'll release an upgrade LND to v0.16.2 shortly. @rockstardev @NicolasDorier

**Comment by Goro2030** [comment]
@dennisreimann , yes, that's the issue definitively. Thanks for connecting the dots!

Since the upgrade to LND v0.16.1, my ARM server with 2GB was surprisingly slow, and I've spent hours trying to find out what was going on ... in the LND thread you've linked, more savvy people found that the LND upgrade was eating up all server memory resources, which is what is going on on my node too. I was about to upgrade the server to 4GB to "solve" for this, but I won't now and wait. Thanks!

**Comment by Goro2030** [comment]
Confirming this bug is trashing the server's disk ... small servers (like mine, 2 CPU, 2GB RAM, 3000 IOPS GP3 AWS instance ) will become extremely unresponsive.

![image](https://user-images.githubusercontent.com/10875290/235313270-af685a79-198d-456b-8e30-1504a78e8ba5.png)

**Comment by pavlenex** [comment]
I believe this one got resolved after we bumped to lnd 0.16.2-beta if not, let us know and we will re-open. Thanks for reporting.

**Comment by Goro2030** [comment]
It is not fully resolved, but the log spam went from hundreds of "unable to fetch transaction" errors, which was trashing the disk, to 4 log entries. So I can call this solved!
